island-is/island.is
# feat(samradsgatt): Add email form to subscription box on case details page

# Add email form to subscription box on details page

## What

Add email subscription box to case details page

## Why

It is missing.

## Screenshots / Gifs

![image](https://user-images.githubusercontent.com/106164521/212292486-64dd66d7-7e67-425a-9b80-1662c5cd7e7f.png)

https://www.figma.com/file/gFSUsFCIMYWRGa94XsmhpT/DEV-Samr%C3%A1%C3%B0sg%C3%A1tt?node-id=68%3A4030&t=4bKNkomPrY9eZAVT-0

## How to test

- Checkout branch feature/samradsgatt/KAM-710--email-form-on-details-page-left-side-column
- Run samradsgatt locally: yarn start samradsgatt
- Navigate to http://localhost:4200/samradsgatt/Cases/id
- Look at the subscription box in the left side bottom corner.
- Should look like the figma design


## Checklist:
